index.html

```diff
@@ -318,41 +318,31 @@ <h3 id="unacceptablebehavior">
           <li>Use of coded language (also known as "dog whistles") used to
           rally support for hate groups or to intimidate vulnerable groups.
           </li>
+          <li>Patronizing language or behavior:
+            <ul>
+              <li>Assuming that people or groups need concepts defined or explained to them. 
+              </li>
+              <li>Assuming that particular groups of people are technically unskilled (e.g., “So easy your grandmother could do it”).
+              </li>
+              <li>Assuming your experience or knowledge on a topic is essential to a discussion, even if it takes discussion off topic or interrupts the flow of the conversation.
+              </li>
+            </ul>
+          </li>
           <li>Microaggressions, which are small comments or questions, either
           intentional or unintentional, that marginalize people by
           communicating hostile, derogatory, or negative beliefs. Examples
           include:
             <ul>
-              <li>Patronizing language or behavior:
-                <ul>
-                  <li>Be aware that, regardless of the speaker's intentions,
-                  some phrases or constructions lead people to expect a
-                  patronizing statement to follow, and avoid such phrases. For
-                  example, beginning an interjection with "Well, actually..."
-                  can set this expectation and be taken as a sign of
-                  disrespect.
-                  </li>
-                  <li>Assuming without asking that particular people or groups
-                  need concepts defined or explained to them. It’s great to be
-                  sensitive to the fact that people may not be familiar with
-                  technical terms you use every day, but assuming that people
-                  are uninformed can come across as patronizing.
-                  </li>
-                  <li>Assuming that particular groups of people are technically
-                  unskilled (e.g., “So easy your grandmother could do it”).
-                  </li>
-                </ul>
-              </li>
               <li>Repeatedly interrupting or talking over someone else.
               </li>
               <li>Feigning surprise at someone’s lack of knowledge or awareness
               about a topic.
               </li>
               <li>The use of racially charged language to describe an
-              individual or thing (such as “thug” or “ghetto”).
```

Comment:
I agree with removing these examples from this particular bullet, but I think it would be helpful to include "thug" elsewhere, as an example of a trigger term, as suggested in PR https://github.com/w3c/PWETF/pull/238 .

I find "thug" to be a good example of a term that historically was neutral -- and still is for many people -- but for some now carries racial overtones.  I think it's helpful to alert people to such examples.